pkgs/by-name/sq/sql-migrate/package.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "sql-migrate";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "rubenv";
+    repo = "sql-migrate";
+    rev = "v${version}";
+    hash = "sha256-l9HW8L9khr+UfxlhH9z2mrzjemlqRbnnM7rPsf9SCtg=";
+  };
+
+  sourceRoot = "${src.name}";
+
+  vendorHash = "sha256-GDM2YAqLkkmfOJShBbPpxS0Zs3zOK4yCfTCjohSGNIM=";
+
+  # Bundle release metadata
+  ldflags = [
+    # Omit the symbol table and debug information.
+    "-s"
+    # Omit the DWARF symbol table.
+    "-w"
+  ];
+
+  doCheck = true;
+
+  meta = with lib; {
+    homepage = "https://github.com/rubenv/sql-migrate";
+    description = "A SQL schema migration tool for Go programs";
+    sourceProvenance = [ sourceTypes.fromSource ];
+    license = licenses.mit;
+    platforms = platforms.unix;
```

Comment:
Just copy-pasting a [comment I saw in another review about `platforms`](https://github.com/NixOS/nixpkgs/pull/291476#pullrequestreview-1907742373):

When you fix the platforms list this way, you are basically saying the upstream developers develop it for that specific platform only. A typical example is the `zsnes` emulator, written in x86 assembly.

I believe this is not the case here. Just remove this line:

```suggestion
```

Here around Nixpkgs we just wait for volunteers. Indeed a lot of people help keeping mpv and emacs on Darwin.

Until then, if ofborg fails then you can add something like

```nix
meta = { 
  ...
  broken = stdenv.isDarwin; # ofborg failed
}
```